projects/ccf-rui/src/app/shared/components/side-selector/side-selector.component.theme.scss

```diff
@@ -0,0 +1,29 @@
+@use '~@angular/material/theming' as *;
+
+@mixin theme($theme) {
+  $foreground: map-get($theme, foreground);
+  $background: map-get($theme, background);
+  $primary: map-get($theme, primary);
+
+  .side-slider {
+    .mat-slide-toggle-bar {
+      background-color: rgba(25, 118, 210, 0.54) !important;
```

Comment:
These colors should come from the theme!! Not hardcoded! Not to mention the use of !important